crates/generate_artifacts/src/reader_ast.rs

```diff
@@ -652,33 +686,39 @@ fn refetched_paths_with_path<TNetworkProtocol: NetworkProtocol>(
                 }
             }
             SelectionTypeContainingSelections::Object(linked_field_selection) => {
-                path.push(
-                    NameAndArguments {
-                        // TODO use alias
-                        name: linked_field_selection.name.item.into(),
-                        arguments: transform_arguments_with_child_context(
-                            linked_field_selection
-                                .arguments
-                                .iter()
-                                .map(|x| x.item.into_key_and_value()),
-                            // TODO this clearly does something, but why are we able to pass
-                            // the initial variable context here??
-                            initial_variable_context,
-                        ),
+                match linked_field_selection.associated_data.field_id {
+                    DefinitionLocation::Client(_) => {
+                        // Do not recurse into selections of client pointers
```

Comment:
should we recurse into the refetch selection set?

I'll merge regardless, doesn't seem like a blocker (since the refetch selection sets for now only have ids)